src/app/pages/information/information.page.scss

```diff
@@ -0,0 +1,7 @@
+.slide-card {
+  width: 100%;
+}
+
+.multiline {
+  white-space: pre-wrap; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/numbersprotocol/capture-lite/pullRequest?prid=6012707)